simplecpp.cpp

```diff
@@ -2426,8 +2426,16 @@ std::map<std::string, simplecpp::TokenList*> simplecpp::load(const simplecpp::To
             continue;
 
         std::ifstream fin(filename.c_str());
-        if (!fin.is_open())
+        if (!fin.is_open()) {
+            if (outputList) {
+                simplecpp::Output err(fileNumbers);
+                err.type = simplecpp::Output::ERROR;
```

Comment:
Please add a new enum constant. The `ERROR` constant is used when `#error` is encountered in the "active" code.

Reply:
I see, I will add a new enum constant.